using_images/sti_images/python.adoc

```diff
@@ -26,27 +37,54 @@ This image comes in two flavors, depending on your needs:
 
 The RHEL 7 image is available through Red Hat's subscription registry via:
 
-****
-`$ docker pull registry.access.redhat.com/openshift/python-33-rhel7`
-****
+----
+$ docker pull registry.access.redhat.com/openshift/python-33-rhel7
+----
 
 *CentOS 7 Based Image*
 
 This image is available on DockerHub. To download it:
 
-****
-`$ docker pull openshift/python-33-centos7`
-****
+----
+$ docker pull openshift/python-33-centos7
+----
 
-To use these images, you can either access them directly from those registries or push them into your OpenShift docker registry.  In addition, it is recommended that you create an ImageStream that points to the image, either in your docker registry or at the external location.  Your OpenShift resources can then reference the ImageStream.  You can find example ImageStream definitions for all the provided OpenShift images https://github.com/openshift/origin/tree/master/examples/image-streams[here].
+To use these images, you can either access them directly from these
+registries, or push them into your OpenShift docker registry. Additionally,
+you can create an ImageStream that points to the image,
```

Comment:
Could link this first instance of "ImageStream" here to `architecture/core_objects/openshift_model.html#imagestream` for context.